site/viamguides/src/environment-variables/environment-variables.md

```diff
@@ -0,0 +1,223 @@
+
+### Install the Python SDK
+
+Install the Viam Python SDK and all required general dependencies.
+
+1. **Install the SDK**: From the command line in your terminal window (inside the activated `.venv` Python environment), install the Viam Python SDK:
+    ```bash
+    pip3 install viam-sdk
+    ```
+
+    > aside negative
+    > Windows is not supported. If you are using Windows, use the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/windows/wsl/install) and install the Python SDK using the preceding instructions for Linux. For other unsupported systems, see [Installing from source](https://python.viam.dev/#installing-from-source).
+
+<!-- ------------------------ -->
+## Using variables in local development
+Duration: 5
+
+### Copy the sample code
+1. **Refer to Python sample code**: In the Viam app, go to the **CONNECT** tab, and select **Python**. 
+  ![Connect tab](assets/Connect.png)
+1. **Show secrets**: The sample code shows you how to authenticate and connect to a machine, as well as some of the methods you can use on your configured components and services. To show your machine’s API key and API key ID in the sample code, toggle **Include secret** on the CONNECT tab’s Code sample page. 
+  ![Code sample](assets/Code.png)
+1. **Hide secrets**: Instead of leaving our sensitive credentials hard coded in this code sample, let's set up our API key and API key ID as environment variables. Toggle **Include secrets** once again, this time to remove the credentials from the code sample. 
+1. **Copy the code**: From the command line in your terminal window, create a new file called `square.py`, and copy the code sample into this file.
+   ```bash
+   touch square.py
+   ```
+### Run the sample code
+1. **Create `.env` file**: From the command line, create a new file called `.env` to store our environment variables to use in local development.
+    ```bash
+    touch .env
+    ```
+1. **Input environment variables**: Add your environment variables to the `.env` file, formatted like the following:
+    ```
+    ROBOT_API_KEY="sijdb24bjnxsmp18mij0hace6smihu0r"
+    ROBOT_API_KEY_ID="9e135013-f3bf-40fc-a0e3-e6fc66a418d7"
+    ```
+1. **Install `python-dotenv` package**: From the command line, still within our virtual environment `.venv`, install the [`python-dotenv`](https://pypi.org/project/python-dotenv/) package to read key-value pairs from an `.env `file and can set them as environment variables.
+    ```bash
+    pip3 install python-dotenv
+   ```
+1. **Load the variables into your environment**: At the top of your `square.py` file, add the following code to load variables into your environment.
+    ```python
+    from dotenv import load_dotenv
+    import os
+
+    load_dotenv()  # Loads the environment variables from the .env file
+
+    robot_api_key = os.getenv('ROBOT_API_KEY')
+    robot_api_key_id = os.getenv('ROBOT_API_KEY_ID')
```

Comment:
It might be personal preference, but many of the relevant Python repos for Viam projects will "CONSTANT_CASE" these variable names to indicate they should remain constant during the lifetime of the program. It should also help reflect that these values came from an environment variable of the same name:

```suggestion
    ROBOT_API_KEY = os.getenv('ROBOT_API_KEY')
    ROBOT_API_KEY_ID = os.getenv('ROBOT_API_KEY_ID')
```